apps/docusaurus/docs/sdks/ts-sdk/type-safe-contract.md

```diff
@@ -28,57 +28,24 @@ First, download the ABI of the Move contract and save it to a TypeScript file. I
     MODULE_NAME=fungible_asset_launchpad
 
     # save the ABI to a TypeScript file
-    echo "export const ABI = $(curl https://fullnode.$NETWORK.aptoslabs.com/v1/accounts/$CONTRACT_ADDRESS/module/$MODULE_NAME | sed -n 's/.*"abi":\({.*}\).*}$/\1/p') as const" > src/utils/abi.ts
+    echo "export const ABI = $(curl https://fullnode.$NETWORK.aptoslabs.com/v1/accounts/$CONTRACT_ADDRESS/module/$MODULE_NAME | sed -n 's/.*"abi":\({.*}\).*}$/\1/p') as const" > abi.ts
```

Comment:
let user move `abi.ts` to anywhere they want, we don't want to provide an opinion here.